site/content/index.md

```diff
@@ -119,16 +119,56 @@ layout: unstyled
   <div className="prose dark:prose-invert mx-auto max-w-2xl px-4 lg:max-w-6xl lg:px-8 xl:px-12">
     <h2 className="text-center">Features</h2>
 
-    Here are some of the cool Flowershow features:
 
-    - CommonMark, GitHub Flavoured Markdown and Obsidian extensions support (e.g. wikilinks) [[syntax|➡ learn more]]
-    - MDX support for custom React components additions [[custom-components|➡ learn more]]
-    - Customizable layouts of your pages [[layouts|➡ learn more]]
-    - Tailwind support for easy styling in-markdown HTML and custom components [[tailwind|➡ learn more]]
-    - Google analytics support [[analytics|➡ learn more]]
-    - Basic site configurations using single JSON file [[basic-config|➡ learn more]]
+Here are some of the cool features that are currently supported by Flowershow, and some that we're intensively working on.
 
-    👷 There is a lot of stuff we're still working on.
+### ✅/🚧 Markdown
+Flowershow supports **CommonMark** and **GitHub Flavoured Markdown (GFM)** syntax, e.g. code blocks, blockquotes, lists, tasks lists and tables.
+
+[[syntax|Learn more ➡]]
```

Comment:
I think we want to have the option in obsidian links to have full links otherwise these weren't render correctly e.g. i think this should be `[[docs/syntax| Learn more ...]]`

Reply:
I think our `remark-wiki-link-plus` is smart enough to handle this.
Basically Obsidian will automatically create a link in form `[[syntax]]` if there is only one page called `syntax`, no matter in what folder it's located. It would be a bummer for Flowershow to work a user would have to manually correct all links to include the full path.
In this case for example `[[syntax|Learn more ➡]]` is properly converted to `<a class="internal" href="/docs/syntax">Learn more ➡</a>`, without me having to explicitly point at the `/docs` dir.

Reply:
@olayway i don't think remark-wiki-link-plus handles that actually. There is a config option in obsidian setting to have it do relative paths from the base of the wiki.

This is actually something we probably need to note in the docs in our wiki links section that people must use absolute path or relative path for Flowershow published links to work.

<img width="864" alt="image" src="https://user-images.githubusercontent.com/180658/182094414-c9fffe78-563b-48e5-ac2a-885a52377a31.png">

Reply:
Hmm, it is strange that it works locally in this case, but ok...
I'll add a short note to the docs.